transformer_engine/common/fused_attn/fused_attn.cpp

```diff
@@ -138,7 +138,8 @@ NVTE_Fused_Attn_Backend nvte_get_fused_attn_backend(
                             && attn_mask_type != NVTE_Mask_Type::NVTE_PADDING_MASK
                             && sm_arch_ == 90)
                         || (bias_type == NVTE_Bias_Type::NVTE_POST_SCALE_BIAS
-                            && sm_arch_ == 90))))
+                            && (cudnn_runtime_version >= 90000
+                                || sm_arch_ == 80 || sm_arch_ == 90)))))
```

Comment:
Is that the right test? Now if you have sm 70 with a recent cudnn, it will be supported. Is that what you want?

Reply:
sm below 80 are covered by the a few line before ([Line 126](https://github.com/NVIDIA/TransformerEngine/blob/25e146ced7f592a4a04f71f2d05af6f3a2e76924/transformer_engine/common/fused_attn/fused_attn.cpp#L126)). `cudnn_runtime_version >= 90000` is for sm86 and sm89 which will be supported after cudnn 9.0.